ibis/backends/polars/__init__.py

```diff
@@ -407,7 +407,17 @@ def compile(
         return translate(node, ctx=self._context)
 
     def _get_sql_string_view_schema(self, name, table, query) -> sch.Schema:
-        raise NotImplementedError("table.sql() not yet supported in polars")
+        import sqlglot as sg
+
+        cte = sg.parse_one(str(ibis.to_sql(table, dialect="postgres")), read="postgres")
```

Comment:
I think this makes `psycopg2` a requirement for the polars backend.  Maybe we want to import-gate this?

Reply:
I moved the import of psycopg2 in the postgres backend to respective call sites. Does that work?

Reply:
That works!